README.md

```diff
@@ -17,80 +17,5 @@ This projects is open source and may be redistributed under the terms of the [Ap
 ### Builds
 ![master build status](https://dev.azure.com/darkloop/DarkLoop%20Core%20Library/_apis/build/status/Open%20Source/Functions%20Authorize%20-%20Pack?branchName=master)
 
```

Comment:
Considering the change in how to correctly set up authentication, I think it would be nice to add a notice at the top for current users of this package.

Something along the lines of:

For users of version 4.0.1 and lower: Azure has made certain changes to the authentication to the Azure Functions Apps which cause certain errors with Azure Portal. Fortunately this is easy to fix! Please see the respective README's of the isolated and in-process for more information.

Reply:
will do

Reply:
I have added this in #51. Let me know if it's clear, thanks for reviewing this